modules/adopter-registration-impl/src/main/java/org/opencastproject/adopter/statistic/Sender.java

```diff
@@ -112,7 +117,7 @@ public void deleteStatistics(String json) throws IOException {
    * @throws IOException General exception that can occur while processing the POST request.
    */
   private void send(String json, String urlSuffix) throws IOException {
-    send(json, urlSuffix, "GET");
+    send(json, urlSuffix, "POST");
```

Comment:
This does not seem to have anything to do with forward proxy support. Is this change accidental?

Reply:
No, it's fixing a bug.  We're sending data, via GET, to an endpoint expecting a POST.  We'll talk about this in the technical meeting today.

Reply:
I've since confirmed that this gets sent as a POST regardless - probably the library is smart enough to correct dumb mistakes like this.  I'd still like to get this into 11.x in case we ever do get another release done there, but it's no longer driving a potential minor release on its own.